tests/test_sample_rayservice_yamls.py

```diff
@@ -285,6 +285,10 @@ def test_zero_downtime_rollout(self, set_up_cluster):
             for cr_event in cr_events:
                 cr_event.trigger()
 
+
+@pytest.mark.skip(
```

Comment:
Ray Serve introduces some breaking changes (see `ray-service.autoscaler.yaml` and `rayservice_ha_test.go`). The autoscaling test logic no longer works.

(I don't read the code. This is just my observation and guess, and the very limited information that I can get from proxy actor's logs)
Based on my observation and guess, Ray Serve will queue pending requests instead of failing them. The pending requests in the queue will affect the request for "signal" to be scheduled. When I set `max_ongoing_requests` to a small number (e.g. 1), the request for "signal" can be scheduled correctly and unlock some "block" requests. However, some pending requests will be scheduled to the replicas immediately, so the replicas can't be scaled down.

https://sourcegraph.com/github.com/ray-project/ray/-/blob/python/ray/serve/_private/replica_scheduler/pow_2_scheduler.py?L701